pkg/agent/manager/manager_test.go

```diff
@@ -424,14 +424,20 @@ func TestSVIDRotation(t *testing.T) {
 	// RUnlock simulates the end of the request (Rotator should rotate SVIDs now)
 	m.GetRotationMtx().RUnlock()
 
-	// If manager's current SVID is not equals to the first one we generated
-	// it means it rotated
 	mockClk.Add(time.Second)
+	// Loop until we detect an SVID rotation was called in separate process
+	util.RunWithTimeout(t, time.Second, func() {
```

Comment:
We've tried to steer away from short test timeout intervals (not all of the have been replaced). It can lead to flaky tests when running under Travis CI. Maybe something larger? 30 seconds, a minute?

Reply:
#theflakeisreal